docs/_documentations/remote-setupregistries.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: docs
+title: Adding a container registry in remote Codewind
+description: Adding a container registry in remote Codewind
+keywords: container, registry, IDE, guidance, image registry, Appsody, Docker, name, push registry, Kubernetes
+duration: 1 minute
+permalink: remote-setupregistries
+type: document
+---
+
+# Adding a container registry in remote Codewind
+
+## Prerequisite: Determining if you need a container registry 
+1\. First, determine if you need to add a container registry. The following scenarios in Codewind require you to specify which image registry is in use:
+- In the following scenario, configure Codewind with an image registry to push the application image to that specific image registry:
+    - Run Codewind on a remote Kubernetes cluster to develop a Codewind style project.
+- In the following scenario, configure Codewind to use the credentials for a specific image registry:
+    - Run Codewind on a remote Kubernetes cluster to develop an Appsody style project.
+    - The image registry for the Appsody stack requires credentials.
+- In the following scenario, from the command line, enter the `docker login` command in the registry before you create the Appsody project:
+    - Run Codewind locally to develop an Appsody style project.
+    - The image registry for the Appsody stack requires credentials. 
+If you do not develop any Codewind style projects, and you use an image registry that does not require credentials, you do not need to specify an image registry.
+2\. If you do need to add a container registry, start and run your IDE.
+
+## Adding a container registry in Codewind
```

Comment:
I know this is taken from the Che page, but it's confusing that the UI prompts call it an Image Registry while the doc calls it a Container Registry.